recipe/meta.yaml

```diff
@@ -2,8 +2,10 @@
 {% set version = "67" %}
 {% set root_url = "https://naif.jpl.nasa.gov/pub/naif/misc/toolkit_N%(version)04d/C"|format(version=version | int) %}
 
-{% set platform_segment = "PC_Linux_GCC_64bit" %}                                      # [linux64 or aarch64]
-{% set sha256 = "60a95b51a6472f1afe7e40d77ebdee43c12bb5b8823676ccc74692ddfede06ce" %}  # [linux64 or aarch64]
+{% set file_ext = "tar.Z" %}  # [not win]
+{% set file_ext = "zip" %}  # [win]
+{% set platform_segment = "PC_Linux_GCC_64bit" %}                                      # [linux64 or aarch64 or ppc64le]
+{% set sha256 = "60a95b51a6472f1afe7e40d77ebdee43c12bb5b8823676ccc74692ddfede06ce" %}  # [linux64 or aarch64 or ppc64le]
```

Comment:
I assume not having a `platform_segment` variable for `ppc64le` was what was at the root of #38 :shrug:

Reply:
ppc64le is for powerpc and is not supported, aarch64 is already supported